hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -239,208 +239,256 @@ param_list: [
     { name: "REPCNT_THRESHOLDS",
       desc: "Repetition count test thresholds register",
       swaccess: "rw",
-      hwaccess: "hro",
+      hwaccess: "hrw",
+      hwext: "true",
+      hwqe:  "true",
       regwen:   "REGWEN",
+      tags: [// Exclude from writes to these field because they are one-way updates
+             "excl:CsrAllTests:CsrExclWrite"]
       fields: [
         { bits: "15:0",
           name: "FIPS_REPCNT_THRESH",
           desc: '''This is the threshold size for the repetition count health test.
                    This value is used in normal mode when entropy is being tested in
                    FIPS/CC compliance mode.
-                   The default value is 32
+                   This register must be written before this module is enabled, and
+                   once written, will only reflect a value lower than the current value.
```

Comment:
Maybe this could be rephrased a little?  Based on the way it's written I can't tell if it means that "only writes lower than the current / pre-enable value will be accepted" or "will always read a value less than what was written pre-enable" ?

Basically I think you want to get the word "write" or "read" into the text (here and below for *_THRESH, since the text repeats)

Reply:
Agreed - it needs to be re-written, and in some cases this repeated text is wrong for the threshold registers going the "other way". I believe I have clean that all up.